app/controllers/application_controller.rb

```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::API
+  include DeviseTokenAuth::Concerns::SetUserByToken
+
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: %i[name email password password_confirmation])
+  end
```

Comment:
It would be better if we override the `DeviseTokenAuth::RegistrationsController` for this purpose as:
`RegistrationsController < DeviseTokenAuth::RegistrationsController`

Take a look at this:
https://stackoverflow.com/questions/47748632/how-to-permit-params-to-a-specific-devise-controller-with-devise-token-auth-gem